modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -117,6 +118,7 @@ public PythonClientCodegen() {
         // map uuid to string for the time being
         typeMapping.put("UUID", "str");
         typeMapping.put("URI", "str");
+        typeMapping.put("BigDecimal", "Decimal");
```

Comment:
This change is unrelated and generates many new files.
If you need this feature please open it in a separate PR.
Python assumes that non integer numbers are floats.

Can you remove this update from here?

Reply:
Done

Reply:
Can you resolve this conversation?